doc/source/whatsnew/v0.9.0.rst

```diff
@@ -41,9 +41,7 @@ API changes
 
    import io
 
-   data = ('0,0,1\n'
-           '1,1,0\n'
-           '0,1,0')
+   data = "0,0,1\n1,1,0\n0,1,0"
```

Comment:
Same comment as above from doc/source/whatsnew/v0.10.0.rst (can this be written on three lines using `"""`)? After this I think this'll be good to go

Reply:
okay, I'll make the changes asap.